cong/kbfp.cc

```diff
@@ -99,6 +99,7 @@ namespace libsemigroups {
   Congruence::KBFP::current_equals(word_t const& w1, word_t const& w2) {
     init();
     if (is_killed()) {
+      // This cannot be reliably tested since it relies on a race condition
```

Comment:
Can you give more details please? In particular, in what circumstances could this return `UNKNOWN`?